### DISCUSSION TITLE
jenkins

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1602] [Q&A] [answered]
![Screenshot 2023-11-14 183403](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/302416ba-5c34-4bdf-8240-09a117a4c6b5)

**Comment by sreedharm07** [comment]
@palsachennaiah 
there is nothing error. your  ip shown jenkins UI.  
you dint given port.
just give 8080 port after your ip, it will show ui
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/0221db27-5095-4dec-afd6-35e581d17258)

**Reply by palsachennaiah** [reply, marked as answer]
Ok I got it, Thank you.
